src/libs/ajax/OAuth2.ts

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+import { Config } from "../config";
+
+export interface OAuthConfig {
+  clientId: string;
+  authorityEndpoint: string;
+}
+
+export const OAuth2 = {
+  getConfig: async (): Promise<OAuthConfig> => getConfig(),
+};
```

Comment:
For what it's worth, as an alternative to namespace objects, modules could export individual methods and consumers could:

```ts
import * as OAuth2 from 'path/to/OAuth2';
```

if they really want the namespace.